chapter05/mdw/Ex-2.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+
+use feature "say";
+
+my @words;
+
+while(<>){
```

Comment:
I can't remember if there is an actual difference or not, but `push @words <>` should get rid of the need for the while loop without any change to how it works, I think, maybe, possibly.....